api/api/src/test/scala/uk/ac/wellcome/platform/api/services/WorksServiceTest.scala

```diff
@@ -269,6 +269,49 @@ class WorksServiceTest
       )
     }
 
+    it("Should simple_query_syntax into account and not fail when it's invalid") {
```

Comment:
nit: we don’t tend to prefix test descriptions with “should”, it’s implicit

Also, “not fail when it’s invalid” describes a lot of possible test! Could we be more specific? Say:

```scala
it("doesn't throw a too_many_clauses error from precedence syntax")
```